README.md

```diff
@@ -139,6 +141,16 @@
 * [Makes](https://github.com/fluidattacks/makes) - A Nix-based CI/CD pipeline framework for building, testing, and releasing projects in any language, from anywhere. 
 * [Standard](https://github.com/divnix/std) - An opinionated Nix Flakes framework to keep Nix code in large projects organized, accompanied by a friendly CLI/TUI optized for DevOps scenarios.
 
+## Continuous Integration
+
+* [Hercules CI](https://hercules-ci.com/) - Non free continuous integration service for Nix.
```

Comment:
Please add a dash between "Non" and "free" (e.g. "Non-free").

Also seems like the kind of project I'd like to put under a "Paid/Non-Free Sevices" section, but that's just me.